src/frequenz/sdk/config/_config_managing.py

```diff
@@ -44,7 +72,11 @@ def __init__(
         """Initialize this instance.
 
         Args:
-            config_path: The path to the TOML file with the configuration.
+            config_paths: The paths to the TOML files with the configuration. Order
```

Comment:
I saw your motivation to have three config files in #65, which make sense. Still having concerns that an arbitrary list of config files can be very difficult to maintain, in particular if a sync'ed version is adding new keys that are not wanted. But I guess we will limit the number of config files through our deployment setup?

Reply:
Yeah, users of this class can decide how many config files to try make sense, we'll have the number we need. This will be determined by the app. There might even be users not using the Frequenz platform that just want to have one and only config file and no overrides.